libs/mails/src/lib/mail-context.ts

```diff
@@ -69,8 +69,9 @@ export class MailContext implements MailContextInterface {
       templateData: data
     })
   }
-
-  async getUserTemplateName(event: UserEvent): Promise<string> {
+  async getUserTemplateName(event: UserEvent, throwOnMissing: true): Promise<string>
```

Comment:
dont think this one is needed as the other one is identical but can also return Promise<null>